dbt/parsely_dpl_redshift/dbt_project.yml

```diff
@@ -0,0 +1,67 @@
+
+# Name your package! Package names should contain only lowercase characters
+# and underscores. A good package name should reflect your organization's
+# name or the intended use of these models
+name: 'parsely'
+version: '0.0.1'
+
+# This setting configures which "profile" dbt uses for this project. Profiles contain
+# database connection information, and should be configured in the  ~/.dbt/profiles.yml file
+profile: 'default'
+
+# These configurations specify where dbt should look for different types of files.
+# The `source-paths` config, for example, states that source models can be found
+# in the "models/" directory. You probably won't need to change these!
+source-paths: ["models"]
+analysis-paths: ["analysis"]
+test-paths: ["tests"]
+data-paths: ["data"]
+macro-paths: ["macros"]
+
+target-path: "target"  # directory which will store compiled SQL files
+clean-targets:         # directories to be removed by `dbt clean`
+    - "target"
+    - "dbt_modules"
+
+# Below are the variables that are available for configuration
+models:
+  parsely:
+    vars:
+      # The name of the table that contains the raw data inside the schema specified in ~/.dbt/profiles.yml file
+      'parsely:events': 'parsely_rawdata'
+      # All timestamps are in UTC. Specifiy the timezone that the data will be reported in. Available options here: https://docs.aws.amazon.com/redshift/latest/dg/time-zone-names.html
+      'parsely:timezone': "'America/New_York'"
+      # Parse.ly standard events. Recommeneded to leave as-is: "('pageview','heartbeat','videostart','vheartbeat')"
+      'parsely:actions': "('pageview','heartbeat','videostart','vheartbeat')"
+      # Update to 'false' if bot traffic should be included and 'true' if bot traffic should be excluded
+      'custom:excludebottraffic': 'false'
+      # The key in the json dictionary passed through the extra_data field: { "key": "value"}
+      'custom:extradata': "'userType'"
```

Comment:
This should be blank as a default, I think.

Reply:
agreed